src/Pim/Bundle/VersioningBundle/spec/Normalizer/Flat/ProductNormalizerSpec.php

```diff
@@ -73,10 +73,10 @@ function it_normalizes_product(
 
         $this->normalize($product, 'flat', [])->shouldReturn(
             [
-                'sku'        => 'sku-001',
```

Comment:
in the export, the array converter are used
for the versioning, we don't care about the order